src/packages/@ncigdc/modern_components/ProjectsCharts/ProjectsCharts.js

```diff
@@ -161,38 +160,53 @@ export default compose(
       .map(({ gene_id: geneId, symbol }) => ({
         symbol,
         gene_id: geneId,
-        onClick: () =>
-          push({
-            pathname: `/genes/${geneId}`,
-            query: {
-              filters: JSURL.stringify(
-                removeFilter(f => f.match(/^genes\./), fmgChartFilters),
-              ),
-            },
-          }),
+        onClick: () => push({
+          pathname: `/genes/${geneId}`,
+          query: {
+            filters: JSURL.stringify(
+              removeFilter(f => f.match(/^genes\./), fmgChartFilters),
+            ),
+          },
+        }),
         tooltips: stackedBarCalculations[geneId].byProject.reduce(
-          (acc, { projectId, percent, count }) => ({
+          (acc, { count, percent, projectId }) => ({
             ...acc,
             [projectId]: (
               <span>
                 <b>
-                  {projectId}:{' '}
+                  {projectId}
+:
+                  {' '}
                   {
                     (projects.find(p => p.project_id === projectId) || {
                       name: '',
                     }).name
                   }
                 </b>
-                <br /> {count.toLocaleString()} Case{count > 1 ? 's' : ''}{' '}
-                Affected<br />
-                {count.toLocaleString()} / {numUniqueCases.toLocaleString()} ({percent.toFixed(2)}%)
+                <br />
+                {' '}
```

Comment:
Wish we could automatically convert these next few lines (and others of the kind, anywhere else) into string interpolations instead.
May have to look into it at ESLint's own repo 🤔